### PR TITLE
Fix table align removing padding from button link

### PR DESCRIPTION
**Pull request by adeliaferreira**
# Description

This PR will solve an alignment issue that initially was reported on System table, but affects image Table too. This was solve removing a unnecessary component on Devices Table and add a css override inside our create link utils method.

Fixes #THEEDGE-3599

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

File: src/Routes/Devices/DeviceTable.js [hunk 1]
```diff
@@ -178,16 +177,7 @@ const createRows = (
...
-                variant="link"
-                target-href={pathToImage}
-                onClick={(e) => {
-                  e.preventDefault();
-                  window.location.href = `${pathToImage}`;
-                }}
-              >
-                {ImageName}
-              </Button>
+              <a href={pathToImage}>{ImageName}</a>
```
Comment (ldjebran) [inline src/Routes/Devices/DeviceTable.js:180]: createLink function is not working fro this case  ?

Comment (adeliaferreira) [reply src/Routes/Devices/DeviceTable.js]: Changed to use createLink, this was an remain test before adjust method. Thanks

**Review by ldjebran — APPROVED** [review]
lgtm

**Review by de1987 — APPROVED** [review]

**Comment by de1987** [comment]
@mergifyio rebase

**Review by acosferreira — APPROVED** [review]
lgtm

**Pull request merged**